cognite/client/utils/_identifier.py

```diff
@@ -384,6 +398,38 @@ def assert_singleton(self) -> None:
         if not self.is_singleton():
             raise ValueError("Exactly one username (string) must be specified")
 
+    @staticmethod
+    def unwrap_identifier(identifier: str | int | dict) -> str | int | InstanceId:
+        if isinstance(identifier, (str, int)):
+            return identifier
```

Comment:
Can `Username` be int? 🤔 
```suggestion
        if isinstance(identifier, str):
            return identifier
```